<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>irc</class>
 <widget class="QMainWindow" name="irc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>415</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferDefault</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Mudlet IRC Client</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="ircWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>1</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="handleWidth">
          <number>3</number>
         </property>
         <widget class="QListView" name="bufferList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
         <widget class="QWidget" name="viewportWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>3</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="rightMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>1</number>
           </property>
           <item>
            <widget class="QTextBrowser" name="ircBrowser">
             <property name="openExternalLinks">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </widget>
         <widget class="QListView" name="userList">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAlias">
   <property name="text">
    <string>Alias</string>
   </property>
  </action>
  <action name="toolButton_add_trigger_menu">
   <property name="text">
    <string>Trigger</string>
   </property>
  </action>
  <action name="actionVariable">
   <property name="text">
    <string>Variable</string>
   </property>
  </action>
  <action name="actionAlias_2">
   <property name="text">
    <string>Alias</string>
   </property>
  </action>
  <action name="toolButton_delete_trigger_menu">
   <property name="text">
    <string>Trigger</string>
   </property>
  </action>
  <action name="actionVariable_2">
   <property name="text">
    <string>Variable</string>
   </property>
  </action>
  <action name="toolButton_add_trigger_group_menu">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionAliases">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aliases</string>
   </property>
  </action>
  <action name="actionTriggers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Triggers</string>
   </property>
  </action>
  <action name="actionVariables">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Variables</string>
   </property>
  </action>
  <action name="actionScript_options">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Script options</string>
   </property>
  </action>
  <action name="toolButton_search_area_2_menu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search results</string>
   </property>
  </action>
  <action name="actionExpert_mode_menu">
   <property name="text">
    <string>Expert mode</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>

